### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by mjoehl** [johngodley/redirection#652]
Hi. I am using your redirection plugin on our site. I really like this plugin.
I wanted to delete all 404 logs with the "Delete all" button at the bottom. Following error occurd:

```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 7.0.24
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Unexpected token D in JSON at position 0
Action: redirection/v1/404 DELETE
Code: 405 Method Not Allowed
Raw: The requested page can not be displayed because an invalid method (HTTP verb) is used for the access.
```

**Comment by johngodley** [comment]
Can you try the 3.1 beta?

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
